Changes.md

```diff
@@ -9,6 +9,7 @@ Improvements
  - Added <kbd>Ctrl-D</kbd> shortcut to duplicate currently viewed image when viewing the output of Catalogue node (#3545).
  - Added enabled/reset controls to the Crop Window Tool.
  - Improved display of the edited plug in the Inspector's pop-up edit windows.
+- Node Editor : Added decoration to collapsed sections when any of their children have non-default values.
```

Comment:
Likewise, this needs moving to a new section.

Reply:
Fixed [here](https://github.com/GafferHQ/gaffer/pull/3895/commits/fbca318157d833a8fa3a28d2d6c4b224e6892ddd).